test/rules/require-matching-function-name.js

```diff
@@ -0,0 +1,72 @@
+var Checker = require('../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-matching-function-name', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+    });
+
+    describe('option value true', function() {
+        beforeEach(function() {
+            checker.configure({ requireMatchingFunctionName: true });
+        });
+
+        // TODO:
```

Comment:
Nope, these todos are more like a roadmap for this particular rule. I can remove it and we'll let the users to decide what should be done next by opening issues on GitHub.